CONTRIBUTING.md

```diff
@@ -230,10 +182,8 @@ Once changie is installed and your PR is created, simply run `changie new` and c
 
```

Comment:
Let's add a blurb about not needing to worry about the version your change will go into or backporting here.  That gets taken care of by our team when we release.

Reply:
Gave this a go, let me know what you think